cmd/edit/ingress/cmd.go

```diff
@@ -155,6 +175,28 @@ func run(cmd *cobra.Command, argv []string) {
 		private = &privArg
 	}
 
+	var lbType *string
+	if cmd.Flags().Changed(lbTypeFlag) {
+		lbType = &args.lbType
+	} else {
+		if interactive.Enabled() {
```

Comment:
Can we put this in https://github.com/openshift/rosa/blob/9a4f60c1dd9fbae4e6f1a41649e2c2ed1142e857/pkg/interactive/helper.go#L103-L104?

We are trying to have the interactive helper for new interactive getters which are reused for more commands

Reply:
I understand but again, we should double check before making such a change. I am using the same pattern used for other parameters of Ingress. If I just move the load balancer type to another file, but other parameters of Ingress stay in the current files, it may not be clean.